WcaOnRails/app/views/layouts/_navigation.html.erb

```diff
@@ -50,7 +50,10 @@
           <ul class="dropdown-menu" role="menu">
             <li><a href="/results/events.php"><%= icon('signal', class: 'fa-fw fa-rotate-90') %> <%= t '.rankings' %></a></li>
             <li><a href="/results/regions.php"><%= icon('trophy', class: 'fa-fw') %> <%= t '.records' %></a></li>
-            <li><a href="<%= persons_path %>"><%= icon('user', class: 'fa-fw') %> <%= t '.persons' %></a></li>
+            <li><a href="<%= persons_path %>"><%= icon('users', class: 'fa-fw') %> <%= t '.persons' %></a></li>
+            <% if current_user&.wca_id? %>
+              <li><%= link_to "#{icon('user', class: 'fa-fw')} #{t '.my_results'}".html_safe, "/results/p.php?i=#{current_user.wca_id}" -%></li>
```

Comment:
We can avoid both `html_safe` and interpolation by making use o `icon` helper abilities =)
```ruby
link_to icon('user', t('.my_results'))
```